.gitlab/bazel/defs.yaml

```diff
@@ -22,6 +22,8 @@
     BAZEL_OUTPUT_USER_ROOT: "$XDG_CACHE_HOME/bazel"
     BAZEL_REPO_CONTENTS_CACHE: "$BAZEL_OUTPUT_USER_ROOT/repo-contents"
     XDG_CACHE_HOME: "$CI_PROJECT_DIR/.cache"
+    GIT_STRATEGY: "clone"
+    GIT_DEPTH: "2"
```

Comment:
Then please consider removing:
```
  variables:
    GIT_DEPTH: 2
```
from the below `.bazel:ext:windows` (the latter is always used in conjunction with the earlier)

Reply:
TBH, I set `2` as an experiment and forgot to remove it... So `1` would be fine!